app-connector-client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "varkes-app-connector-client",
-  "version": "1.3.23",
+  "version": "1.3.24",
```

Comment:
Currently the prow pipeline will bump the version number, so it will release 1.3.25 and that number must be used then in the examples